claim-169/Readme.md

```diff
@@ -0,0 +1,139 @@
+# CBOR Identity Data in QR-Code
+
+**Tag**: 169 (identity-data)
+
+**Data Item**: JSON Object
+
+**Semantics**: Unique Identity Data of a Citizen in QR-Code
+
+**Point of contact**: Resham Chugani (resham@mosip.io)
+
+# 1.Introduction
+
+This document specifies a generic data structure and encoding mechanisms for storing Unique Identity Data of a Citizen registered using any ID platforms. It also specifies a transport encoding mechanism in a machine-readable optical format (QR).
+
+# 2.Rationale
+
+Once a citizen is registered in an identity system, their data serves as the foundation for identification, granting them access to social benefits and government services. The level of assurance in this identification process varies depending on the authentication method employed. Low assurance is achieved through the use of basic identifiers like ID numbers, demographic data, passwords, or PINs. Conversely, higher assurance levels are attained through methods such as one-time passwords (OTP) or biometrics.
+
+Among these methods, biometric based authentication, such as facial authentication, offers the highest level of assurance as it directly verifies the presence of the citizen. While this is effective for online systems where verification is conducted on a server, offline authentication presents challenges in maintaining a similarly high level of assurance.
```

Comment:
And also works for people with no phone. Can we add this?

Reply:
Done